### PR TITLE
Handle multiple invitation addresses

### PR DESCRIPTION
**Pull request by EmiM**
…e addresses

### CODE REVIEW

File: packages/desktop/src/renderer/sagas/invitation/handleInvitationCode.saga.test.ts [hunk 1]
```diff
@@ -1,87 +1,87 @@
...
+//           args: {
+//             title: 'Invalid link',
+//             subtitle: 'The invite link you received is not valid. Please check it and try again.',
+//           },
+//         })
+//       )
+//       .not.put(communities.actions.createNetwork(payload))
+//       .run()
+//   })
+// })
```
[the remark this answers is not in the document]
Comment (EmiM) [reply packages/desktop/src/renderer/sagas/invitation/handleInvitationCode.saga.test.ts]: Restored 👍🏻

File: packages/desktop/src/renderer/sagas/invitation/handleInvitationCode.saga.ts [hunk 2]
```diff
@@ -32,26 +31,27 @@ export function* handleInvitationCodeSaga(
...
-  )
+  // yield* put(
+  //   modalsActions.openModal({
+  //     name: ModalName.warningModal,
+  //     args: {
+  //       title: 'Invalid link',
+  //       subtitle: 'The invite link you received is not valid. Please check it and try again.',
+  //     },
+  //   })
+  // )
```
[the remark this answers is not in the document]
Comment (EmiM) [reply packages/desktop/src/renderer/sagas/invitation/handleInvitationCode.saga.ts]: Restored 👍🏻

File: packages/desktop/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.test.tsx [hunk 3]
```diff
@@ -202,24 +203,14 @@ describe('join community', () => {
...
-      InviteLinkErrors.InvalidCode,
-    ],
+    [`http://${validCode}`, InviteLinkErrors.InvalidCode],
+    // ['QmZoiJNAvCffeEHBjk766nLuKVdkxkAT7wfFJDPPLsbKSE=bbb', InviteLinkErrors.InvalidCode],
+    // ['bbb=y7yczmugl2tekami7sbdz5pfaemvx7bahwthrdvcbzw5vex2crsr26qd', InviteLinkErrors.InvalidCode],
+    // ['QmZoiJNAvCffeEHBjk766nLuKVdkxkAT7wfFJDPPLsbKSE= ', InviteLinkErrors.InvalidCode],
+    // ['nqnw4kc4c77fb47lk52m5l57h4tc', InviteLinkErrors.InvalidCode],
+    // [`https://otherwebsite.com/${Site.JOIN_PAGE}#${validCode}`, InviteLinkErrors.InvalidCode],
+    // [`${QUIET_JOIN_PAGE}?param=nqnw4kc4c77fb47lk52m5l57h4tcxceo7ymxekfn7yh5m66t4jv2olad`, InviteLinkErrors.InvalidCode],
+    // [`${Site.MAIN_PAGE}/share?${validCode}`, InviteLinkErrors.InvalidCode],
```
[the remark this answers is not in the document]
Comment (EmiM) [reply packages/desktop/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.test.tsx]: Restored

File: packages/state-manager/src/sagas/identity/identity.slice.ts [hunk 4]
```diff
@@ -8,6 +8,7 @@ import {
   type Identity,
   type RegisterCertificatePayload,
   type StoreUserCertificatePayload,
+  SaveCSRPayload,
```
[the remark this answers is not in the document]
Comment (EmiM) [reply packages/state-manager/src/sagas/identity/identity.slice.ts:11]: Unused anyway here

File: packages/state-manager/src/sagas/communities/communities.selectors.test.ts [hunk 5]
```diff
@@ -97,32 +97,32 @@ describe('communitiesSelectors', () => {
...
+  //   const { store } = prepareStore()
+  //   const factory = await getFactory(store)
+  //   await factory.create<ReturnType<typeof communitiesActions.addNewCommunity>['payload']>('Community', {
+  //     registrarUrl: code,
+  //     port: 0,
+  //     onionAddress: '',
+  //   })
+  //   const invitationUrl = communitiesSelectors.invitationUrl(store.getState())
+  //   expect(invitationUrl).toEqual(invitationShareUrl(code))
+  // })
```
[the remark this answers is not in the document]
Comment (EmiM) [reply packages/state-manager/src/sagas/communities/communities.selectors.test.ts:125]: We won't support any old format

**Review by Kacper-RF — APPROVED** [review]

**Pull request merged**